### DISCUSSION TITLE
Issue with deleting files/folder in the MobiusSync folder

### DISCUSSION THREAD

**Discussion by TAutoWelt** [MobiusSync/MobiusSync#161] [Q&A]
I have been using Mobius Sync for a while with no issues... until now, when I updated my iPhone to the new iOS 16.
Not sure if this was caused by the update but now I have problems deleting files from the MobiusSync folder using the native Files app on the iPhone. 
When I try to delete any file or folder i get the error from the screenshot. 
![IMG_3230](https://user-images.githubusercontent.com/114222005/191925512-2df1c919-ff39-4b2b-8020-306754a594aa.jpg)

This only happens in the Mobius Sync folder.
When I delete some files on another device that synchronizes with my iPhone the files are removed from the iPhone when MobiusSync process the synching. But I can't manually delete files from my iPhone.
I tried restarting the app, the phone but nothing changes. 

Looking forward for the solution of this issue,
Thanks

**Comment by SimonPickup** [comment]
I cannot reproduce this. Anyone else?

**Comment by SimonPickup** [comment]
This seems to be an iOS issue: https://www.macobserver.com/tips/how-to/fix-you-dont-have-permission-to-access-trash-iphone-and-ipad/
